Update Aspire documentation for clarity and accuracy (#8322)
Particular/docs.particular.net: platform/aspire/index.md

- [ServiceControl](/servicecontrol) error, audit, and monitoring instances
- [ServicePulse](/servicepulse)
- A managed RavenDB database for ServiceControl, or one supplied by the AppHost
- The configured message [transport](#supported-components-transport)
- The platform license

> Transport and licensing are configured once on the platform resource and propagated to every component, so the containers start in the correct order. ServiceControl's persistence is configured on the same resource and shared by the Error and Audit instances that require it. [NServiceBus endpoints](/nservicebus/endpoints/) attach with `WithParticularPlatform(...)` and pick up the same transport connection string and license without additional wiring, and configure their own persistence separately.

Reviewer: There have been struggles with getting customers to realize that the ServiceControl database should actually be treated like a database, so it might be better to emphasize that and avoid the word "persistence" altogether when talking about ServiceControl needs.
```suggestion
Transport and licensing are configured once on the platform resource and propagated to every component, so the containers start in the correct order. ServiceControl's database is configured on the same resource and shared by the Error and Audit instances that require it. [NServiceBus endpoints](/nservicebus/endpoints/) attach with `WithParticularPlatform(...)` and pick up the same transport connection string and license without additional wiring, and configure their own persistence separately.
```

| [PostgreSQL](/transports/postgresql/)                     | Not yet supported            |
| [IBM MQ](/transports/ibmmq/)                              | Not yet supported            |

> ### ServiceControl persistence

Reviewer: ```suggestion
### ServiceControl database
```

snippet: aspire-quick-start-endpoint

> These defaults are intended for local development. For deployment scenarios, configure an explicit transport, ServiceControl persistence, and license as described in [Configuring the transport](#configuring-the-transport), [Configuring ServiceControl persistence](#configuring-servicecontrol-persistence), and [Configuring the license](#configuring-the-license).

Reviewer: ```suggestion
These defaults are intended for local development. For production deployments, configure an explicit transport, ServiceControl database, and license as described in [Configuring the transport](#configuring-the-transport), [Configuring ServiceControl database](#configuring-servicecontrol-database), and [Configuring the license](#configuring-the-license).
```

| `S3BucketForLargeMessages` (`IExpressionValue?`) property on `AmazonSQSTransportSettings`              | Optional |
| `QueueNamePrefix`          (`string?`) property on `AmazonSQSTransportSettings`                        | Optional |

## Configuring ServiceControl database

ServiceControl uses a database to store error and audit data, retry state, and saga audit history. This is ServiceControl's own database; it is separate from any persistence your NServiceBus endpoints use for sagas, outbox, or subscriptions, which you configure in each endpoint as usual. See [Supported components](#supported-components) for the persisters currently wired through the integration.

> Configure persistence on the platform resource, then pass the resulting persistence builder into the ServiceControl Error and Audit instances that need it. The Monitoring instance does not require a database. Follow the [managing database instance guidance](/servicecontrol/ravendb/containers.md) when setting up for production use.

Reviewer 2: ```suggestion
Configure persistence on the platform resource, then pass the resulting persistence builder into the ServiceControl Error and Audit instances that need it. The Monitoring instance does not require a database. Follow the [managing database guidance](/servicecontrol/ravendb/containers.md) when setting up for production use.
```

| `S3BucketForLargeMessages` (`IExpressionValue?`) property on `AmazonSQSTransportSettings`              | Optional |
| `QueueNamePrefix`          (`string?`) property on `AmazonSQSTransportSettings`                        | Optional |

> ## Configuring ServiceControl persistence

Reviewer: ```suggestion
## Configuring ServiceControl database
```

Reviewer 2: Happy to replace all these but we do use the word "persistence" when talking about SC db on our docs, eg: https://docs.particular.net/servicecontrol/servicecontrol-instances/#persistence

Reviewer: Agreed, that's gonna happen. I don't know if I like it there either, but it's more important to avoid the word here where we're talking about both concepts together on the same page. Thanks for making the changes!

> ServiceControl uses a persistence backend to store error and audit data, retry state, and saga audit history. This is ServiceControl's own database; it is separate from any persistence your NServiceBus endpoints use for sagas, outbox, or subscriptions, which you configure in each endpoint as usual. See [Supported components](#supported-components) for the persisters currently wired through the integration.

Reviewer: ```suggestion
ServiceControl uses a database to store error and audit data, retry state, and saga audit history. This is ServiceControl's own database; it is separate from any persistence your NServiceBus endpoints use for sagas, outbox, or subscriptions, which you configure in each endpoint as usual. See [Supported components](#supported-components) for the persisters currently wired through the integration.
```
